lib/ansible/modules/cloud/amazon/kms_facts.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This Ansible library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this library.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: kms_facts
+short_description: Gather facts about AWS KMS keys
+description:
+    - Gather facts about AWS KMS keys including tags and grants
+version_added: "2.4"
+author: "Will Thames (@willthames)"
+options:
+  filters:
+    description:
+      - A dict of filters to apply. Each dict item consists of a filter key and a filter value.
+        The filters aren't natively supported by boto3, but are supported to provide similar
+        functionality to other modules. Standard tag filters (C(tag-key), C(tag-value) and
```

Comment:
`C()` is for option values. Here `filters` is the key so `tag-key` is part of the value (but also part of a key in the value). It's arguable, I'd say.